### DISCUSSION TITLE
有什么办法把投掷物准星和切换左右手绑到一个键上吗

### DISCUSSION THREAD

**Discussion by CudtMFrag** [Purple-CSGO/CSGO-Config-Presets#16] [Q&A] [answered]
如果是
`bind  "mouse4"           "+crosshair_throw;toggle cl_righthand";`
投掷物准星是正常的，但是只能从左手变右手，右手换左手没反应，控制台提示
`"cl_righthand" = "110" ( def. "1" ) client archive ss                            - Use right-handed view models.`
如果是
`bind  "mouse4"           "toggle cl_righthand;+crosshair_throw";`
可以正常切换左右手，但是按过一次后准星就变不回来了

**Comment by Purple-CSGO** [comment, marked as answer]
首先把这两个功能放在一个键很奇怪，投掷物准星也有全自动版（见[文档](https://config.upup.cool/v1/optional#_3-%E5%8F%AA%E4%BD%BF%E7%94%A8%E3%80%90%E8%87%AA%E5%8A%A8%E6%8A%95%E6%8E%B7%E7%89%A9%E5%87%86%E6%98%9F%E3%80%91%E4%BB%85%E5%85%BC%E5%AE%B9%E5%9F%BA%E6%9C%AC%E5%8F%8C%E9%94%AE%E8%B7%B3%E6%8A%95)）

如果你想实现，这里的 + - 指令有一个特性，就是复合指令中 `+指令` 后面的不会执行，但是你可以自定 `+指令` 

```bash
bind "mouse4" "+xhand";
alias "+xhand" "exec crosshair_throw; toggle cl_righthand";
alias "-xhand" "exec crosshair";
```

P.S. 为啥有 `+crosshair_throw` ？很奇怪，如果是 `exec crosshair_throw` 那就不影响了

**Reply by CudtMFrag** [reply]
因为基本投掷物准星那用的是`+crosshair_throw`，我直接复制过去的
